scalardl-test/src/main/java/scalardl/Common.java

```diff
@@ -81,4 +81,9 @@ public static Retry getRetryWithExponentialBackoff(String name) {
 
     return Retry.of(name, retryConfig);
   }
+
+  public static void setIntervalAndMaxRetry(Config config){
```

Comment:
I think it's a better idea to pass the parameters (maxRetries, sleepBaseMillis) to `getRetryWithExponentialBackoff()` like the following:
```java
  public static Retry getRetryWithExponentialBackoff(String name, long sleepBaseMillis, int maxRetries) {
    IntervalFunction intervalFunc = IntervalFunction.ofExponentialBackoff(sleepBaseMillis, 2.0);
    RetryConfig retryConfig =
        RetryConfig.custom().maxAttempts(maxRetries).intervalFunction(intervalFunc).build();

    return Retry.of(name, retryConfig);
  }
```

I think you can refer to the code around the following code:
https://github.com/scalar-labs/kelpie-test/blob/c7279de7f53cb75a713befc13721874282cda51d/scalardb-test/src/main/java/kelpie/scalardb/Common.java#L149-L161

It is for `getRetryWithFixedWaitDuration()`, but you can do the same for the method.

Reply:
Thank you for the suggestion.
I will check and make necessary changes.

Reply:
I have made changes similar to the suggestion by passing retry count and interval sleep time to the `getRetryWithExponentialBackoff()` function